docs/_source/guides/llms/practical_guides/set_up_annotation_team.md

```diff
@@ -20,22 +20,29 @@ Here's how you can do this:
 1. Get the list of users who will be annotating:
 
 ```python
-import httpx
 import argilla as rg
 
 rg.init(
     api_url="...",
     api_key="..."
 )
 
+# get the list of users
+# optional: filter users to get only those with annotator role
+users = [u for u in rg.User.list() if u.role == "annotator"]
+```
+
+```{note}
+If you are using a version earlier than 1.11.0 you will need to call the API directly to get the list of users. Note that, in that case, users will be returned as dictionaries and so `users.username` will be `users['username']` instead.
+```python
 # make a request using your Argilla Client to get the list of users
 rg_client= rg.active_client().client
 auth_headers = {"X-Argilla-API-Key": rg_client.token}
 http=httpx.Client(base_url=rg_client.base_url, headers=auth_headers)
 users = http.get("/api/users").json()
 
-# optional: filter users to get only those with annotator role
-users = [u for u in users if u["role"] == "annotator"]
+# filter users to get only those with annotator role
+users = [user for user in users if user['role']=='annotator']
```

Comment:
Here I think we should be consistent with the double-quotes usage, as we're doing in most of the cases, correct me if I'm wrong @davidberenstein1957 but we should advocate for " > ', right?

Reply:
Totally my bad! I have the bad habit of not being consistent with the quotes. Changing it now!